config/src/main/java/org/springframework/security/config/annotation/web/configurers/ott/OneTimeTokenLoginConfigurer.java

```diff
@@ -321,12 +320,8 @@ private <C> C getBeanOrNull(H http, Class<C> clazz) {
 		if (context == null) {
 			return null;
 		}
-		try {
-			return context.getBean(clazz);
-		}
-		catch (NoSuchBeanDefinitionException ex) {
-			return null;
-		}
+
+		return context.getBeanProvider(clazz).getIfUnique();
```

Comment:
Same comment here about `getIfAvailable`

Reply:
It's also same `getIfUnique` comment